blockchain/blockchain.go

```diff
@@ -69,7 +69,7 @@ const (
 )
 
 const (
-	triesInMemory = 4
+	DefaultTriesInMemory = 4
```

Comment:
Do you have any reason to change the scope of the variable? `DefaultTriesInMemory` -> `defaultTriesInMemory`